.github/dependabot.yml

```diff
@@ -677,3 +677,7 @@ updates:
       - dependency-name: "pytmv1"
     schedule:
      interval: daily
+  - package-ecosystem: pip
```

Comment:
please remove the entry for 
directory: /docker/pytmv1
note that you didnt add pytmv1 to be ignored so dependabot will autoupdate this now

Reply:
done!